packages/cli/src/oclif/commands/invoke.js

```diff
@@ -972,7 +1025,8 @@ class InvokeCommand extends BaseCommand {
           if (filePath === '-') {
             inputStream = process.stdin;
           } else {
-            inputStream = fs.createReadStream(filePath, { encoding: 'utf8' });
+            const fd = await fs.open(filePath);
+            inputStream = fd.createReadStream({ encoding: 'utf8' });
```

Comment:
Interesting, is this an edge case? I definitely recalled testing the input file option and it worked ok

Reply:
I changed `require('node:fs')` to `require('node:fs/promises')` in the `auth start` [PR](https://github.com/zapier/zapier-platform/pull/872), so it was working and then was broken by the `auth start` PR.